packages/library-base/src/commonMain/kotlin/io/realm/dynamic/DynamicRealmObject.kt

```diff
@@ -144,6 +146,13 @@ public interface DynamicRealmObject : BaseRealmObject {
     public fun getObjectList(propertyName: String): RealmList<out DynamicRealmObject>
 }
 
+public fun DynamicRealmObject(type: String, vararg properties: Pair<String, Any?>): DynamicMutableRealmObject {
```

Comment:
According to discussions I added `DynamicMutableRealmObject.create` instead of this. We can tweak if we find the above pattern better in the future, but for now just avoiding it to ensure that we don't have diverging patterns. If this is really annoying to write people can just do the extension themselves.